_posts/2018-11-08-google-cpp-style-guide-is-no-good.md

```diff
@@ -0,0 +1,297 @@
+---
+layout: post
+title: Google C++ Style Guide is No Good
+date: '2018-11-06T20:59:00.000-07:00'
+author: Eugene Yakubovich
+tags:
+- C++
+modified_time: '2018-10-23T20:59:46.484-07:00'
+---
+
+The [Google Style Guide (GSG)](https://google.github.io/styleguide/cppguide.html) for C++ has become popular outside of Google.
+But I don't like it at all. There... I said it.
+I want to take this opportunity to explain why I feel so strongly about it.
+
+## GSG "style"
+The general observation is that the style guide is written in a prohibitive fashion.
+Unlike C++ Core Guidelines that try to explain how to use the language effectively, GSG is about forbidding the use of certain features.
+Many of the rules prohibit the use of a feature over the fear of confusion, abuse, ambiguity and bugs.
+It is a well known fact that every feature (in every language) can be abused and misunderstood.
```

Comment:
maybe add a reference?

Reply:
I'll add quote to where Bjarne said that.